sdk/schemaregistry/Azure.Data.SchemaRegistry/src/SchemaFormat.cs

```diff
@@ -20,9 +20,15 @@ public SchemaFormat(string value)
         }
 
         private const string AvroValue = "Avro";
+        private const string JsonValue = "JSON";
+        private const string CustomValue = "Custom";
 
         /// <summary> Avro Serialization schema type. </summary>
         public static SchemaFormat Avro { get; } = new SchemaFormat(AvroValue);
+        /// <summary> Json Serialization schema type. </summary>
+        public static SchemaFormat Json { get; } = new SchemaFormat(JsonValue);
```

Comment:
Sounds like we should use JSON schema content type here.

Reply:
Not sure if this update is exactly what you meant, so let me know if I need to adjust it